web/src/app/rotations/UserForm.tsx

```diff
@@ -1,9 +1,15 @@
 import React from 'react'
-import { PropTypes as p } from 'prop-types'
 import { FormContainer, FormField } from '../forms'
 import { UserSelect } from '../selection'
 
-export default function UserForm(props) {
+interface UserFormProps {
+  errors?: Array<Error>
+  onChange?: (value: { users: string[] }) => void
+  disabled?: boolean
+  value: { users?: Array<string> }
```

Comment:
Myabe we should create a `UserFormValue` type to ensure these are identical

Reply:
Ok! Updated version with a separate `UserFormValue` is pushed